Mailer/MailerInterface.php

```diff
@@ -35,4 +35,22 @@ public function sendConfirmationEmailMessage(UserInterface $user);
      * @return void
      */
     public function sendResettingEmailMessage(UserInterface $user);
+
+    /**
+     * Define the email address used to send confirmation emails from.
+     *
+     * @param string|null $email
+     *
+     * @return void
+     */
+    public function setFromEmailConfirmation($email);
```

Comment:
I don't agree with you : FOSUB is in alpha stage of a major version, so if you can't do it now, when are you supposed to ? This modification is also the result of an identified issue ... Nevermind, I've changed the way to inject these values